src/components/text/text.component.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled'
+
+import { InputVariant } from '../../types/common.type.ts'
+
+export const H3 = styled('h3')`
+  color: var(--ui-color-black-60);
+
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+`
+
+export const InputPlaceholder = styled('div')<{ variant: InputVariant }>`
```

Comment:
<{ variant: InputVariant }> 

maybe we can extract this to common type and reuse